### PR TITLE
documentation drive: document what was undocumented

### PR DESCRIPTION
**Pull request by dominictarr**
this PR finishes documenting all the undocumented methods.

addUnboxer and unbox definitely need more work.
also, I noticed there are several methods (about getting the latest message) that do not seem to be in use anymore, they should probably be removed.

### CODE REVIEW

**Comment by dominictarr** [comment]
Also, some methods are available to the local api but not over muxrpc. sometimes this is necessary because muxrpc can't do things like observables, other times I just forgot to put it in the `api.md` each method should be explicit about wether it's exposed or not.

**Comment by dominictarr** [comment]
closes: https://github.com/ssbc/ssb-db/issues/256

File: README.md [hunk 1]
```diff
@@ -356,26 +356,127 @@ which is why it starts with `_`.
...
-* status            - get status information about index building
-* version           - current ssb-db version
-* queue             - append a message to write queue (without writing immediately)
-* post              - get called back when a message is appended
-* since             - observable of the current offset of underlying flumelog.
-* addUnboxer        - add a handler for unboxing (decrypting) messages
-* box               - encrypt a message content
+### getAtSequence ([id, seq], cb(err, msg))
+
+get a message at a given feed `id` with given `sequence`.
```
Comment (arj03) [inline README.md]: maybe: "for a given feed id" instead of "at a given feed" or is my english bad? :-)

File: README.md [hunk 2]
```diff
@@ -356,26 +356,127 @@ which is why it starts with `_`.
...
+calls back a message or an error, takes a two element array
+with a feed `id` as the first element, and `sequence` as second element.
+
+needed for [ssb-ebt replication](https://github.com/ssbc/ssb-ebt)
+
+### getVectorClock (cb)
+
+load a map id to latest sequence (`{<id>: <seq>,...}`) for every feed in the database.
```
Comment (arj03) [inline README.md]: load a map of

**Comment by arj03** [comment]
Added minor wording suggestions, looks good otherwise!

File: README.md (hunk 1 above)
Comment (dominictarr) [reply README.md]: no your english is better than mine

**Comment by dominictarr** [comment]
merged into 19.1.1

**Pull request merged**